### DISCUSSION TITLE
Como obter o selo de patrocinador no github?

### DISCUSSION THREAD

**Discussion by HugoHenriql** [Schweinepriester/github-profile-achievements#798] [General]
Somente queria saber

**Comment by Artostapyshyn** [comment]
@HugoHenriql Help to get big brain achievement, accept this answer

**Comment by Artostapyshyn** [comment]
@HugoHenriql Help to get big brain achievement, accept this answer

**Comment by peguimasid** [comment]
Hello @HugoHenriql, you must sponsor a user on Github that automatically the achievement comes to you. You can make any donation, the minimum I tested was $1. You can choose who you want to sponsor at this link: https://github.com/sponsors/explore.

Please mark the answer as correct if you think it's correct, if you have any more questions feel free to ask me :)
